local.env.dist

```diff
@@ -0,0 +1,13 @@
+# ATLAS_TOKEN      - Terraform Cloud access token ($TFC_API_TOKEN)
+# onepasswordkey   - 1Password service account access key
+# ORGANIZATION     - Name of the Terraform Cloud organization to be backed up
+# VAULT_NAME       - 1Password vault name  "Apps Dev"
+# VAULT_ITEM       - Item within the 1Password vault  "Terraform Workspace Backups"
+# BACKUP_BASE_NAME - Base name of the backup file (${BACKUP_BASE_NAME}-yyyy-mm-dd.tar.gz)  "tfc-backup"
+
+ATLAS_TOKEN=
+onepasswordkey=
```

Comment:
Why is this lowercase and not word-separated? I don't see it referenced anywhere.

Reply:
I changed the variable's name and forgot to update it here.